Updated referenced docker images

r0ssing opened the pull request:
- Bumped postgres and nginx versions.
- Removed the specific branch when cloning sync-endpoint (= this can be controlled by setting default branch on sync-endpoint repo)
- Updated README.md

permission-error committed:
Update README.md

permission-error committed:
Update docs

wbrunette committed:
Merge pull request #50 from permission-error/docs-patch3

Update README.md

r0ssing committed:
Updated postgres to v.12.17

r0ssing committed:
Updated nginx to v.1.25.4

r0ssing committed:
Removed branch spec. when cloning sync-endpoint

r0ssing committed:
Update README.md

Updated example to Ubuntu 22.04, fixed broken link

wbrunette commented on init-odkx-sync-endpoint.py:
```diff
@@ -148,7 +148,7 @@ def run_docker_builds():
 
 def run_sync_endpoint_build():
-    os.system("git clone -b master --single-branch --depth=1 https://github.com/odk-x/sync-endpoint ; \
+    os.system("git clone --single-branch --depth=1 https://github.com/odk-x/sync-endpoint ; \
```
if you don't specify the branch, won't it default to master?

r0ssing replied:
It will default to the branch defined as "default branch" under the settings tab in the repo in GH... (so currently yes; master. But then we can easily change it to main later)

wbrunette replied:
but don't we want it pointing at circleci-taketwo?

r0ssing committed:
Specifying circleci-taketwo branch

Remember to change this (ie. remove -b <branch spec>) when merging to develop/main

r0ssing commented:
Hi @wbrunette - I've updated the checkout command to checkout the taketwo branch. This PR still targets the development branch of sync-endpoint-default-setup though - should we create an "updatejdk"-branch in this repo as well and redirect this PR to that?

wbrunette commented:
a personal opinion, I think it's fine, evetntually all update-jdk branches will be moved to development as we will likely just have "development" and "main" but happy to have consistency though in the short term with the upgrade-jdk

r0ssing committed:
Updated bootstrap.sh

Fixed some errors due to updated versions
(timeout no longer accepts "-t", postgresql no longer accepts storing unencrypted passwords)

r0ssing commented:
Upon further testing I noticed some edits had to be made to db-bootstrap (encrypting stored password, updated 'timeout' command). @wbrunette: can we merge this to develop branch now?

Commit:
pre-lua

Commit:
pre-lua

r0ssing committed:
Add unzip proxy

r0ssing committed:
Update nginx config to support new version

r0ssing committed:
Add helper shell scripts
